dlc-sled-storage-provider/src/lib.rs

```diff
@@ -361,6 +361,8 @@ impl Storage for SledStorageProvider {
                 },
             )
         .map_err(to_storage_error)?;
+
+        channel_tree.flush().map_err(|e| Error::StorageError(e.to_string()))?;
```

Comment:
nit: can you use `to_storage_error` (just to be consistent with the rest of the code)